build/build-image/Dockerfile

```diff
@@ -15,9 +15,9 @@
 # This file creates a standard build environment for building Kubernetes
 FROM gcr.io/google_containers/kube-cross:KUBE_BUILD_IMAGE_CROSS_TAG
 
-ENV http_proxy=KUBE_BUILD_HTTP_PROXY \
-  https_proxy=KUBE_BUILD_HTTPS_PROXY \
-  no_proxy=KUBE_BUILD_NO_PROXY
+ENV http_proxy=KUBERNETES_HTTP_PROXY \
```

Comment:
do we still need these? I don't think we're doing any network access inside the build image anymore.